src/main/resources/init_db.sql

```diff
@@ -1,20 +1,45 @@
-CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
-USE `taxi_service`;
+CREATE SCHEMA IF NOT EXISTS `taxi_service_base` DEFAULT CHARACTER SET utf8;
+USE `taxi_service_base`;
 
-DROP TABLE IF EXISTS `manufacturers`;
-CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+DROP TABLE IF EXISTS `taxi_service_base`.`manufacturers`;
+CREATE TABLE `taxi_service_base`.`manufacturers` (
+                    `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                    `name` VARCHAR(225) NOT NULL,
+                    `country` VARCHAR(225) NOT NULL,
+                    `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                    PRIMARY KEY (`id`));
 
-DROP TABLE IF EXISTS `drivers`;
-CREATE TABLE `drivers` (
-                                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
-                                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+DROP TABLE IF EXISTS `taxi_service_base`.`drivers`;
+CREATE TABLE `taxi_service_base`.`drivers` (
+                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                  `name` VARCHAR(225) NOT NULL,
+                  `license_number` VARCHAR(225) NOT NULL,
+                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                  PRIMARY KEY (`id`),
+                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
+                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+DROP TABLE IF EXISTS `taxi_service_base`.`cars`;
+CREATE TABLE `taxi_service_base`.`cars` (
```

Comment:
I don't think we need t specify a schema name for the queries since we have USE `taxi_service_base`;

Reply:
took away